.github/workflows/release.yml

```
@@ -152,13 +164,16 @@ jobs:
       - name: Build WASM runtime
         id: srtool_build
         uses: chevdor/srtool-actions@v0.8.0
+        env:
+          BUILD_OPTS: --release "${{ needs.setup.outputs.build_options }}"
```

Needs this patch: 

```suggestion
          BUILD_OPTS: "--release ${{ needs.setup.outputs.build_options }}"
```

https://github.com/gluwa/creditcoin3/commit/705910e62c9d39a07e0bd2245931dc3fe37d3a52